packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -0,0 +1,204 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const error = require('../error');
+const isA = require('joi');
+const ScopeSet = require('fxa-shared').oauth.scopes;
+const validators = require('./validators');
+
+module.exports = (log, db, config, customs, push, oauthdb, subhub) => {
+  // Skip routes if the subscriptions feature is not configured & enabled
+  if (! config.subscriptions || ! config.subscriptions.enabled) {
+    return [];
+  }
+
+  const SUBSCRIPTIONS_MANAGEMENT_SCOPE =
+    'https://identity.mozilla.com/apps/subscriptions';
```

Comment:
design nit: since this is an "internal" scope used by FxA itself, I wonder if it should be something like "/acct/subscriptions" rather than "/apps/subscriptions" for a bit of conceptual separation.  I don't think we really have any prior art for this though.

Reply:
> I wonder if it should be something like "/acct/subscriptions" rather than "/apps/subscriptions" for a bit of conceptual separation. I don't think we really have any prior art for this though.

I left some thoughts [over here](https://github.com/mozilla/fxa/issues/882#issuecomment-485750529).